acceptancetests/assess_wallet.py

```diff
@@ -123,7 +123,7 @@ def assert_set_wallet(client, name, limit, error_strings):
     try:
         _try_setting_wallet(client, name, limit)
     except JujuAssertionError as e:
-        if error_strings['pass'] not in e.message:
+        if error_strings['pass'] not in str(e):
```

Comment:
Is this correct, does `JujuAssertionError` implement `__str__` correctly?

Reply:
JujuAssertionError inherits from the built-in AssertionError (which I believe should implement that). It has no other visible fields in the docs so I hope it does :D